src/videojs-http-streaming.js

```diff
@@ -698,7 +698,7 @@ if (videojs.registerPlugin) {
   videojs.plugin('reloadSourceOnError', reloadSourceOnError);
 }
 
-module.exports = {
+export default {
```

Comment:
this should be equivalent but can be moved to a separate PR.

Reply:
I think it's fine to have this change in this PR